client/sections.js

```diff
@@ -6,12 +6,6 @@ const sections = [
 		group: 'root',
 		enableLoggedOut: true,
 	},
-	{
-		name: 'sites',
-		paths: [ '/sites' ],
```

Comment:
Loving the new dashboard FWIW. One thing I was curious about, though, was the old `my-sites` module. Should we get rid of that completely, now that we're removing the section? 

Also, related to the above, this section was handling the `/sites/:site` route:

https://github.com/Automattic/wp-calypso/blob/0b1198ce5237a2540aaeeff024da4d5532a84acb/client/my-sites/index.js#L8

Should the sites dashboard be handling that route, and have it redirect to `/home/:site` for example?

Reply:
Yes I think you're right. I didn't think of removing it because I thought the other sections in the `my-site` group might need a parent module or something. But rethinking about it I don't see why they would. And good spotting on the need to retain the redirect to `/sites/:site` to My Home.

`/my-sites/controller.js` will still exist, but it'll now just be shared middleware to help other sections implement their own routes.

The naming is going to start getting confusing I fear. `/my-sites/README.md` talks about the "My Sites" section of Calypso, which has always meant "the main site-specific Calypso area". But now we have this new dashboard a lot of folks have said "My Sites" more naturally feels like it refers to the dashboard.

That's a job for the next project when we adjust the navigation and the site switcher. Just musing out loud.